src/Kernel.php

```diff
@@ -55,7 +52,7 @@ public function getLogDir(): string
         return dirname(__DIR__).'/var/log';
     }
 
-    public function registerBundles(): iterable
+    public function registerBundles()
```

Comment:
As the demo project requires PHP 7.1, you could keep this

Reply:
Yes, but I prefer to use the exact same code from the Symfony recipe. That's why we're also reverting the changes in our config (see #666).

Reply:
"PHP 5.5.9 or higher" is described in the `README.md` as a requirement.

Reply:
well, the readme was forgotten when changing the requirement in the composer.json

Reply:
There's a pending PR for that: #625.